sources/vega/commands/v1/transaction.proto

```diff
@@ -84,4 +84,13 @@ message Transaction {
   // A version of the transaction, to be used in the future in case changes are implemented
   // to the Transaction format.
   uint32 version = 2000;
+
+  // Proof of work contains the random transaction id used by the client and the nonce. 
+  ProofOfWork pow = 3000;
```

Comment:
Should we create a nerw transaction version as well, or just assume that if the pow is nil, the wallet didn't specify a pow.

I think it would be easier if at first we support both format of transaction until we deprecate it compeltely.

Reply:
done.